src/pages/Sessions.jsx

```diff
@@ -1,10 +1,13 @@
 import React, { useEffect, useState } from "react";
-import { Typography } from "@material-ui/core";
+import { Button, Typography } from "@material-ui/core";
+import { Add } from "@material-ui/icons";
 
 import SessionAPI from "../api/SessionAPI";
+import RegistrationDialog from "../components/registration/RegistrationDialog";
 
 function Sessions() {
   const [sessions, setSessions] = useState([]);
+  const [displayRegDialog, setDisplayRegDialog] = React.useState(false);
```

Comment:
why are we explicitly calling useState from React here, but not on the line above?

Reply:
oof good catch, updated!